Form/Type/FormatterType.php

```diff
@@ -212,11 +215,6 @@ public function configureOptions(OptionsResolver $resolver)
             'choices' => $formatters,
         ];
 
-        // NEXT_MAJOR: Remove the method_exists hack when dropping support for symfony < 2.7
-        if (count($formatters) > 1 && method_exists('Symfony\Component\Form\AbstractType', 'configureOptions')) {
```

Comment:
The first part of the if is no longer needed? NEXT_MAJOR comment only talks about removing `method_exists`

Reply:
Don't know why we should only add this option when we have at least one formatter.

Reply:
https://github.com/sonata-project/SonataFormatterBundle/pull/211#issuecomment-240460367

Here is the discussion, not sure if it applies now, but seems like when we have one formatter, is displayed as a hidden input, and on SF 2.8 atleast it does not have `choice_translation_domain`

Reply:
Fixed